doc/config/Doxyfile.template(1)

```diff
@@ -0,0 +1,2484 @@
+############################################################################
```

Comment:
do we really need this file? Isn't this simply a replicate of Doxyfile.template?